cluster/addons/any-volume-datasource/crd/hello-populator-crd.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: hellos.hello.k8s.io
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+  annotations:
+    api-approved.kubernetes.io: unapproved
```

Comment:
should we really be installing new unapproved API types?
this seems like it should either be approved or in x-k8s.io

Reply:
that should definitely be updated there... needing to add "unapproved" is a good sign this shouldn't be using this suffix

Reply:
a test-only fixture could use something like `example.com`

Reply:
Fixing in https://github.com/kubernetes-csi/lib-volume-populator/pull/23 and will update this PR once it is merged.

Reply:
The change in Hello populator was merged and I'm waiting for the container image to be updated. 
As for the change in this PR, I've tested locally with below change by using locally built image and confirmed that it works. I will update this PR once the container image is updated.
https://github.com/mkimuram/kubernetes/commit/c150d2564fd369a3394805cc08a58d76f36cd76c

Reply:
The updated image has been published as v1.0.1, so I've updated this PR as I [mentioned](https://github.com/kubernetes/kubernetes/pull/108664#discussion_r841095990). So, this PR should be ready to be merged.

/hold cancel